utils/m0reportbug

```diff
@@ -886,6 +895,20 @@ optparse() {
             --addb) COLLECT_ADDB_P=1;;
             --skip-addb-files) SKIP_ADDB_FILE_P=1;;
             -b|--binaries) COLLECT_BINARIES_P=1;;
+            --coredumps) 
+                if [[ -n "$2" && ($2 == "true" || $2 == "false")]];
+                then
+                    coredump=$(echo $2 | sed -e 's/,/\n/g')
+                else
+                    echo "$2 reqire argument" && usage
+                fi
+                if [ $coredump == "true" ];
+                then
+                    COLLECT_COREDUMPS_P=1
+                else
+                    COLLECT_COREDUMPS_P=0
+                fi
+                shift;;
```

Comment:
Use "--coredumps" option  in same fashion  as "--binaries, --addb, --skip-addb-files" are used in m0reportbug. 

Simply set "OLLECT_COREDUMPS_P=1" if "--coredumps" is specified on command line, do not check "true || false"